core/vm/instructions.go

```diff
@@ -859,14 +858,12 @@ func makeLog(size int) executionFunc {
 		}
 
 		d := scope.Memory.GetCopy(int64(mStart.Uint64()), int64(mSize.Uint64()))
-		interpreter.evm.StateDB.AddLog(&types.Log{
-			Address: scope.Contract.Address(),
-			Topics:  topics,
-			Data:    d,
-			// This is a non-consensus field, but assigned here because
-			// core/state doesn't know the current block number.
```

Comment:
can we preserve this information by adding a comment for `func (s *StateDB) AddLog...`?

Reply:
Yup, good idea